mpv-plugin/SimpleHistory/SimpleHistory-2023.09.25.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+USE_MPV="rdepend"
+MPV_REQ_USE="lua"
+inherit mpv-plugin
+
+DESCRIPTION="Continue watching last played, manage and play from history"
+HOMEPAGE="https://github.com/Eisa01/mpv-scripts#simpleundo"
+SRC_URI="https://github.com/Eisa01/mpv-scripts/archive/refs/tags/25-09-2023.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/mpv-scripts-25-09-2023"
+
+LICENSE="BSD-2"
+KEYWORDS="~amd64"
+
+MPV_PLUGIN_FILES=(
+	SimpleHistory.lua
+)
+
+src_prepare() {
+	mv scripts/SimpleHistory.lua .
+
+	default
+}
+
+pkg_postinst() {
+	mpv-plugin_pkg_postinst
+	einfo "h to see the history, ctrl-r to resume previously closed"
+	einfo "Documentation https://github.com/Eisa01/mpv-scripts#simplehistory"
```

Comment:
It should install the readme, not link to it via url.

Docs should also include the [example `.conf`](https://github.com/Eisa01/mpv-scripts/blob/master/script-opts/SimpleHistory.conf) file.

Underneath `MPV_PLUGIN_FILES` you can add `DOCS=( script-opts/SimpleHistory.conf README.md )` which probably should work.

Reply:
Thanks for the review.

Looks like in [mpv-plugin](https://github.com/gentoo/guru/blob/master/eclass/mpv-plugin.eclass) no variable for `script-opts`? Adding conf to `MPV_PLUGIN_FILES` results in invalid `/etc/mpv/scripts/script-opts/SimpleHistory.conf`, when must be `/etc/mpv/script-opts/SimpleHistory.conf`

Reply:
Thanks for the review - added DOCS.

I love `einfo` with some short advices and a link, this is useful for people.